packages/celotool/src/lib/env-utils.ts

```diff
@@ -140,6 +140,7 @@ export enum DynamicEnvVar {
   ORACLE_AWS_CLUSTER_REGION = 'ORACLE_{{ oracleContext }}_AWS_KUBERNETES_CLUSTER_REGION',
   ORACLE_AZURE_SUBSCRIPTION_ID = 'ORACLE_{{ oracleContext }}_AZURE_SUBSCRIPTION_ID',
   ORACLE_AZURE_TENANT_ID = 'ORACLE_{{ oracleContext }}_AZURE_TENANT_ID',
+  ORACLE_CLOUD_PROVIDER = '', // Will be automatically set in oracle
```

Comment:
This feels a little dirty, was there an issue with the `Omit` approach?

Reply:
Was having troubles with it earlier, but just figured it out! Next commit will show the new implementation.